go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/openshift/rosa
 
-go 1.20
+go 1.21
```

Comment:
@tirthct If you require ROSA to be using `1.21` of go, then that should be done under a separate JIRA first so that we can ensure the CI pipeline is working as expected. If not, then please revert this change to stick with `1.20`.

cheers,

Rob

Reply:
Reverted back to 1.20